integration_client.py

```diff
@@ -3,14 +3,126 @@
     CoverageRecord,
     DataSource,
     ExternalIntegration,
+    PresentationCalculationPolicy,
+)
+from core.coverage import (
+    CatalogCoverageProvider,
+    WorkCoverageProvider,
 )
-from core.coverage import CatalogCoverageProvider
 from core.metadata_layer import (
     Metadata,
     ReplacementPolicy,
 )
 
-class IntegrationClientCoverageProvider(CatalogCoverageProvider):
+
+class WorkPresentationCoverageProvider(WorkCoverageProvider):
```

Comment:
This isn't the ideal file for these classes, but for now I like having the IntegrationClientCoverageProvider separate and this avoids looped imports with `coverage.py`.